<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdressBook</class>
 <widget class="QDialog" name="AdressBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Адресная книга</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/png/24x24/Address book.png</normaloff>:/images/images/png/24x24/Address book.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="6">
      <widget class="QCheckBox" name="checkBoxZK3">
       <property name="text">
        <string>Заказчик</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QCheckBox" name="checkBoxRp6">
       <property name="text">
        <string>Ремонтник</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Информация</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6" colspan="2">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item row="3" column="7">
      <widget class="QCheckBox" name="checkBoxTh4">
       <property name="text">
        <string>Тех. обсл</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QCheckBox" name="checkBoxMn5">
       <property name="text">
        <string>Монтажник</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Тип</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/png/24x24/Create.png</normaloff>:/images/images/png/24x24/Create.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/png/24x24/Delete.png</normaloff>:/images/images/png/24x24/Delete.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Приминить</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/images/png/24x24/Apply.png</normaloff>:/images/images/png/24x24/Apply.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QWidget" name="widget" native="true"/>
     </item>
     <item row="0" column="0" rowspan="6" colspan="3">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="6" column="6" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdressBook</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdressBook</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
